News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/adapter/ViewHolder.java

```diff
@@ -304,6 +328,16 @@ public void setRssItem(RssItem rssItem) {
         }
     }
 
+    private static void scaleTextSize(TextView tv, int initialSize) {
+        SharedPreferences mPrefs = PreferenceManager.getDefaultSharedPreferences(tv.getContext());
+        float scalingFactor = Float.parseFloat(mPrefs.getString(SettingsActivity.SP_FONT_SIZE, "1.0"));
+        if(initialSize < 0) {
+            initialSize = Math.round(tv.getTextSize());
+        }
+        int textSize = (int)Math.round(initialSize*scalingFactor/3.5);  // factor 3.5 is there to be able to use scaled pixels
```

Comment:
Where does the factor `3.5` come from? 

Maybe we can use something like this to convert `px` to `sp`:
`float sp = px / tv.getContext().getResources().getDisplayMetrics().scaledDensity;`

Reference: https://stackoverflow.com/a/42715556